datahub-web-react/src/app/analytics/event.ts

```diff
@@ -519,6 +520,12 @@ export interface EmbedProfileViewInDataHubEvent extends BaseEvent {
     entityUrn: string;
 }
 
+export interface EmbedLookupNotFoundEvent extends BaseEvent {
+    type: EventType.EmbedLookupNotFoundEvent;
+    url: string;
+    reason: 'NO_ENTITY_FOUND' | 'MULTIPLE_ENTITIES_FOUND';
```

Comment:
Totally up for debate 😆

Reply:
i think these are good! nit: can we make these strings constants to reuse?

Reply:
I think I'd prefer a type we can import vs a constant. That way we get compiler support, and we can still do refactoring shortcuts in vscode.